product-pricing-objects/src/main/kotlin/com/hedvig/productPricingObjects/dtos/ExtraBuildingDto.kt

```diff
@@ -0,0 +1,12 @@
+package com.hedvig.productPricingObjects.dtos
+
+import com.hedvig.productPricingObjects.objects.ExtraBuildingType
+import java.util.UUID
+
+data class ExtraBuildingDto(
```

Comment:
Since under the dto package
```suggestion
data class ExtraBuilding(
```